googleprediction/gp-connector/gp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/googleprediction-config/component.xml

```diff
@@ -1,4 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+   Copyright (c) 2005-2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Fixed in latest repo https://github.com/wso2-extensions/esb-connector-googleprediction/blob/master/src/main/resources/googleprediction-config/component.xml